src/goo/geometrypack/text/TextMeshGenerator.js

```diff
@@ -402,11 +402,11 @@ function meshesForText(text, font, options) {
 			meshBuilder.addMeshData(meshData, transform);
 		}
 	
-		if (options.backface)
-		  frontFace();
-      		else // If the back face shouldn't be visible, set extrusion to 0s
+		if (options.backface) {
+			frontFace();
```

Comment:
Shouldnt it be the backface()  that is drawn if options.backface is true?

Also, could you make the default behavior be to draw backface if nothing is specified (to behave like before), by doing options.backface = options.backface !== undefined ? options.backface || true; at the top?